ops/op2/signature.rs

```diff
@@ -337,6 +444,9 @@ fn stringify_token(tokens: impl ToTokens) -> String {
     .map(|s| s.to_string())
     .collect::<Vec<_>>()
     .join("")
+    // Ick.
+    // TODO(mmastrac): Should we pretty-format this instead?
+    .replace(" , ", ", ")
```

Comment:
Probably not worth it - every time I got a compile error the spaces around didn't bother me - it was readable enough